github/event_types.go

```diff
@@ -149,6 +149,12 @@ type EditChange struct {
 	Title *struct {
 		From *string `json:"from,omitempty"`
 	} `json:"title,omitempty"`
+	Name *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"title,omitempty"`
+	Note *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"title,omitempty"`
 	Body *struct {
 		From *string `json:"from,omitempty"`
 	} `json:"body,omitempty"`
```

Comment:
Look at the description of `EditChange`:

```Go
// EditChange represents the changes when an issue, pull request, or comment has
// been edited.
type EditChange struct {
```

I'm not sure if it makes the most sense to keep reusing it for project changes, project column changes and project card changes. That becomes a lot of different things all for one common struct, isn't it? (Maybe it's fine, not sure...)

I was considering creating a new struct instead...

But now I'm starting to think, maybe it'd be simpler/better to just inline the struct into the event?

That way, each `changes` field is going to contain exactly the fields that are supposed to be there, nothing more. I think it'd be better API.

It would be harder/more verbose to create new instances of these structs, but I'm not sure if that's something people do often, is it?

This is what it'd look like:

```Go
// ProjectEvent is triggered when project is created, modified or deleted.
// the webhook event name is "project".
//
// GitHub docs https://developer.github.com/v3/activity/events/types/#projectevent
type ProjectEvent struct {
	...
	Changes *struct{
		Name *struct {
			From *string `json:"from,omitempty"`
		} `json:"name,omitempty"`
		Body *struct {
			From *string `json:"from,omitempty"`
		} `json:"body,omitempty"`
	} `json:"changes,omitempty"`
	...
}

// ProjectCardEvent is triggered when a project card is created, updated, moved, converted to an issue, or deleted.
// the webhook event name is "project_card".
//
// GitHub docs: https://developer.github.com/v3/activity/events/types/#projectcardevent
type ProjectCardEvent struct {
	...
	Changes *struct{
		Note *struct {
			From *string `json:"from,omitempty"`
		} `json:"note,omitempty"`
	} `json:"changes,omitempty"`
	...
}

// ProjectColumnEvent is triggered when a project column is created, updated, moved, or deleted.
// the webhook event name is "project_column".
//
// GitHub docs https://developer.github.com/v3/activity/events/types/#projectcolumnevent
type ProjectColumnEvent struct {
	...
	Changes {
		Name *struct {
			From *string `json:"from,omitempty"`
		} `json:"name,omitempty"`
	} `json:"changes,omitempty"`
	...
}
```

What do you think?